src/components/PostPageDetail/index.js

```diff
@@ -610,6 +610,7 @@ const PostPageDetailIdComponent = (props) => {
                   onPressScore={handleOnPressScore}
                   onPressBlock={() => refBlockComponent.current.openBlockComponent(item)}
                   isSelf={profile.myProfile.user_id === item.actor?.id}
+                  isShowDM
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

A new prop `isShowDM` has been added to the component. However, no value is provided for this prop. If it's a boolean prop and its presence implies true, then this is fine. Otherwise, you should provide a value for it.

```diff
+                   isShowDM
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->